### PR TITLE
ADD: stale workflow

### PR DESCRIPTION
**Pull request by Segelzwerg**

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+  runs-on: ubuntu-latest
+  steps:
+  - name: Close Stale PRs
+    uses: actions/stale@v5.0.0
+    with:
+      remove-pr-stale-when-updated: true
```
Comment (iTitus) [inline .github/workflows/stale.yml:13]:
```suggestion
```

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      days-before-issue-close: -1
+      days-before-pr-stale: 30
```
Comment (iTitus) [inline .github/workflows/stale.yml:15]: why 30?

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+jobs:
+  stale:
+  runs-on: ubuntu-latest
+  steps:
+  - name: Close Stale PRs
+    uses: actions/stale@v5.0.0
+    with:
+      remove-pr-stale-when-updated: true
+      days-before-issue-close: -1
```
Comment (iTitus) [inline .github/workflows/stale.yml:14]: why not close issues?

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/stale.yml (hunk 2 above)
Comment (Segelzwerg) [reply .github/workflows/stale.yml:14]: 60 is too long, anything else you prefer?

File: .github/workflows/stale.yml (hunk 3 above)
Comment (Segelzwerg) [reply .github/workflows/stale.yml:13]: We have some longstanding issue and would be tedious to reopen them every now and than

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Close Stale PRs
+      uses: actions/stale@v5
+      with:
+        days-before-issue-close: -1
+        days-before-pr-stale: 30
+        exempt-pr-assignees: 'iTitus, Segelzwerg'
```
Comment (iTitus) [inline .github/workflows/stale.yml:15]:
```suggestion
        exempt-assignees: 'iTitus, Segelzwerg'
```

File: .github/workflows/stale.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Close Stale PRs
+      uses: actions/stale@v5
+      with:
+        days-before-issue-close: -1
```
Comment (iTitus) [inline .github/workflows/stale.yml:13]:
```suggestion
```

File: .github/workflows/stale.yml [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+        days-before-pr-stale: 30
```
Comment (iTitus) [inline .github/workflows/stale.yml:14]:
```suggestion
        days-before-stale: 30
```

File: .github/workflows/stale.yml (hunk 5 above)
Comment (iTitus) [reply .github/workflows/stale.yml:13]: all assigned issues are exempt either way.

File: .github/workflows/stale.yml (hunk 6 above)
Comment (iTitus) [reply .github/workflows/stale.yml:14]: too narrow

File: .github/workflows/stale.yml (hunk 4 above)
Comment (iTitus) [reply .github/workflows/stale.yml:15]: too narrow

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/stale.yml (hunk 5 above)
Comment (Segelzwerg) [reply .github/workflows/stale.yml:13]: But some issues haven't been assigned yet and still are long standing

Comment (iTitus) [reply .github/workflows/stale.yml:13]: then they are stale per definition :D

Comment (Segelzwerg) [reply .github/workflows/stale.yml:13]: Yes kinda. That's why I wanted to exclude issues for now.

File: .github/workflows/stale.yml (hunk 3 above)
Comment (Segelzwerg) [reply .github/workflows/stale.yml]: @iTitus how do we proceed here? Let's start with PR staling and leave the issues for now?

**Comment by iTitus** [comment]
Still 3 open conversations. Either unassigned issues become stale or this makes no sense at all.

File: .github/workflows/stale.yml (hunk 5 above)
Comment (Segelzwerg) [reply .github/workflows/stale.yml:13]: @iTitus I wouldn't close them automatically, but marking them `Stale` is fine. Is that a compromise we could agree on?

Comment (iTitus) [reply .github/workflows/stale.yml:13]: yes

Comment (iTitus) [inline .github/workflows/stale.yml:13]:
```suggestion
        days-before-close: -1
```

Comment (iTitus) [reply .github/workflows/stale.yml:13]: Too narrow

**Review by iTitus — CHANGES_REQUESTED** [review]

File: .github/workflows/stale.yml (hunk 5 above)
Comment (Segelzwerg) [reply .github/workflows/stale.yml:13]: I thought we wanted to keep issues open and only mark them stale?

Comment (iTitus) [reply .github/workflows/stale.yml:13]: yes and the same for prs, no?

Comment (Segelzwerg) [reply .github/workflows/stale.yml:13]: ah no let's close them, if they are no changes made after a certain time.

Comment (iTitus) [reply .github/workflows/stale.yml:13]: ok 👍

**Review by iTitus — APPROVED** [review]

**Pull request merged**